elastic/kibana
# [Observability] [Exploratory View] rename Analyze data labels to Explore data

## Summary

Fixes https://github.com/elastic/kibana/issues/116279

Updates entry points for Uptime, User Experience, and APM to use `Explore data` label.

<img width="509" alt="Screen Shot 2021-11-09 at 2 53 29 PM" src="https://user-images.githubusercontent.com/11356435/140995807-4dd773b9-24bd-49e1-9a6c-4e4a68e68e82.png">

### Testing
1. Connect Kibana to edge oblt data via the `oblt-cli` tool
2. Navigate to `app/apm/services/elastic-co-frontend/overview?comparisonEnabled=true&comparisonType=day&environment=ENVIRONMENT_ALL&kuery=&rangeFrom=now-15m&rangeTo=now&transactionType=page-load`
3. Ensure that the apm entry point for exploratory view reads `Explore data`
4. Navigate to `app/ux?rangeFrom=now-15m&rangeTo=now&percentile=50&serviceName=elastic-co-frontend`
5. Ensure that the user experience entry point for exploratory view reads `Explore data`
6. Navigate to `app/uptime/`
7. Enusre that the uptime entry point for exploratory view reads `Explore data`
